src/java.base/share/classes/jdk/internal/vm/TranslatedException.java

```diff
@@ -35,10 +37,8 @@
 import java.util.zip.GZIPInputStream;
 import java.util.zip.GZIPOutputStream;
 
-import jdk.vm.ci.common.JVMCIError;
-
 /**
- * Support for translating exceptions between different runtime heaps.
+ * Support for translating exceptions between the HotSpot heap and libjvmci heap.
  */
 @SuppressWarnings("serial")
 final class TranslatedException extends Exception {
```

Comment:
Would it be possible to re-format this to avoid the wildly long (150+) lines? This seems to be moving jdk.vm.ci.hotspot.TranslatedException and hard to see what is going on.

Reply:
Is there some tool support for formatting Java source code to meet OpenJDK coding guidelines? Rather unfortunate that one has to do this manually and reviewers have to spend time manually checking it.

Reply:
There isn't a code or agreed conventions. The cleanup in the latest version looks okay.